pom.xml

```diff
@@ -78,7 +78,7 @@
     <properties>
         <!-- The oldest Java version supported by this tool.               -->
         <!-- Will control compilation of the main code and the unit tests. -->
-        <base.java.version>1.8</base.java.version>
+        <base.java.version>11</base.java.version>
```

Comment:
I think it'd better to migrate compiler plugin to use `release` flag instead of `source/target` as well.

Reply:
The PR is from January ... it should be merged anyway, then continue with further changes.

Reply:
> I think it'd better to migrate compiler plugin to use `release` flag instead of `source/target` as well. (Regardless of `base.java.version` being 8 or 11.)

I retract my request. I've only now learnt that where appropriate `release` is used already.